thaomonster/ACNH-Collections
# Feature/villager list

### What does this PR do? 
>Clearly and concisely describe the feature. 

This PR displays the API information on the main page. It also styles the main page.

### Is this a feature or Fix/Refactor?  
Feature

### Problems Encountered & Resulting Solutions  
>State any problems encountered 
>Describe your code changes in detail for reviewers.

None.

### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?
`src/Components/App/App.js` lines: 3-4, 11-14, 19;
`src/Components/VillagerList/VillagerList.js` lines: 1-32;
`src/Components/VillagerList/_VillagerList.scss` lines: 1-35;

### How should this be tested?
>What steps need to be taken to confirm functionality? 
1. `git fetch` branch
2. run `npm start` 
3. Watch the page load with all characters

### Steps To Take
>Any questions and possible steps moving forward.

Create a loading page.

### This PR is related to which issues:
>Tag issue here: `Closes #<issue_number>`
Closes #3
